File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/sysinfo/SysInfoItem.java
package com.didichuxing.doraemonkit.kit.sysinfo;

/**
 * Created by wanglikun on 2018/9/14.
 */

public class SysInfoItem {
    public boolean isPermission;
    public final String name;
    public final String value;


    public SysInfoItem(String name, String value) {
        this.name = name;
        this.value = value;
    }

    public SysInfoItem(String name, String value, boolean isPermission) {
        this.name = name;
        this.value = value;
        this.isPermission = isPermission;
    }
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/sysinfo/SysInfoItem.kt
package com.didichuxing.doraemonkit.kit.sysinfo

/**
 * @author lostjobs created on 2020/6/9
 */
class SysInfoItem(
  val type: Int = SysInfoItemAdapter.TYPE_SECTION,
  val title: String,
  val value: String? = null,
  var isPermission: Boolean = false
)